http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -431,13 +432,19 @@ class DataprocInterpreter[F[_]: Parallel](
         LeoLenses.cloudContextToGoogleProject.get(params.runtimeAndRuntimeConfig.runtime.cloudContext),
         new RuntimeException("this should never happen. Dataproc runtime's cloud context should be a google project")
       )
-      _ <- googleDataprocService.stopCluster(
-        googleProject,
-        region,
-        DataprocClusterName(params.runtimeAndRuntimeConfig.runtime.runtimeName.asString),
-        Some(metadata),
-        params.isDataprocFullStop
-      )
+      _ <- googleDataprocService
+        .stopCluster(
+          googleProject,
+          region,
+          DataprocClusterName(params.runtimeAndRuntimeConfig.runtime.runtimeName.asString),
+          Some(metadata),
+          params.isDataprocFullStop
+        )
+        .recoverWith { case _: com.google.api.gax.rpc.PermissionDeniedException =>
+          logger
+            .info(ctx.loggingCtx)(s"Leo SA can't access the project. ${googleProject} might've been deleted.")
+            .as(None)
```

Comment:
we're recovering here so that Leo will eventually move the runtime to `Error` instead of having it stuck in a `Stopping` status